buildSrc/src/main/java/com/google/firebase/gradle/plugins/GmavenVersionChecker.kt

```diff
@@ -24,6 +24,10 @@ import org.gradle.api.tasks.TaskAction
 
 abstract class GmavenVersionChecker : DefaultTask() {
 
+  companion object {
+    private val PERMITTED_TO_LEAVE_BETA = setOf("")
```

Comment:
nit: we could initialize with an empty `Set` rather than a `Set` with a single empty `String` element :)

```suggestion
    private val PERMITTED_TO_LEAVE_BETA = setOf<String>()
```